sdk/storage/storage-blob/src/ServiceURL.ts

```diff
@@ -57,11 +58,23 @@ export class ServiceURL extends StorageURL {
    *                            pipeline, or provide a customized pipeline.
    * @memberof ServiceURL
    */
-  constructor(url: string, pipeline: Pipeline) {
-    super(url, pipeline);
+  // constructor(url: string, pipeline: Pipeline) {
+  //   super(url, pipeline);
+  //   this.serviceContext = new Service(this.storageClientContext);
+  // }
+  /**
+   * Creates an instance of ServiceURL.
+   *
+   * @param {string} connectionString
+   * @param {Pipeline} pipeline Call StorageURL.newPipeline() to create a default
+   *                            pipeline, or provide a customized pipeline.
+   * @memberof ServiceURL
+   */
+  constructor(connectionString: string, pipeline: Pipeline) {
```

Comment:
We should be creating the default pipeline with info from the connection string. Since the constructor overload is still under discussion, let's add a static method for now
```
static function fromConnectionString(connectionString: string): ServiceURL {
  const extractedCreds = extractPartsWithValidation(connectionString);
  const pipeline = StorageURL.newPipeline(extractedCreds);
  super(extractedCreds.url, pipeline);
}